src/Common/GeneratorField.php

```diff
@@ -30,9 +30,21 @@ class GeneratorField
     public $inIndex = true;
     public $isNotNull = false;
 
-    public function parseDBType($dbInput)
+    /**
+     * @param Column $column
+     * @param $dbInput
+     */
+    public function parseDBType($dbInput, $column = null)
     {
         $this->dbInput = $dbInput;
+        if (!is_null($column)) {
+            if ($column->getLength() > 0) {
+                $this->dbInput = $this->dbInput.','.$column->getLength();
```

Comment:
instead of using two if statement we can just use ternary,

```
$this->dbInput = ($column->getLength() > 0) ? $this->dbInput.','.$column->getLength() : $this->dbInput;
$this->dbInput = (!$column->getNotnull()) ? $this->dbInput = $this->dbInput.':nullable' : $this->dbInput;```